components/UploadedFilePreview.js

```diff
@@ -182,13 +189,12 @@ UploadedFilePreview.propTypes = {
   onClick: PropTypes.func,
   border: PropTypes.string,
   size: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.array]),
-  maxHeihgt: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.array]),
+  maxHeight: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.array]),
   /** If true, a link to the original file will be added if possible */
   hasLink: PropTypes.bool,
 };
 
 UploadedFilePreview.defaultProps = {
-  size: 88,
```

Comment:
Places using a `StyledDropzone` don't provide this prop. Is it a breaking change for them?

Reply:
@Betree : Actually realized we don't need to remove this default value. I was doing some testing on other stuff and was removing this value to refactor the code and then forgot to add it back. Sorry. I've reverted this change. 

Will merge if there's no objections. 😄